src/api/events.js

```diff
@@ -41,18 +40,24 @@ function bindEvent (elem, event, handler) {
   elem.addEventListener(name, handler, capture);
 }
 
-function bindEvents (elem, events) {
+export default function (elem, events) {
+  var queue = [];
+  var ready = false;
+
   Object.keys(events).forEach(function (name) {
-    bindEvent(elem, name, events[name]);
+    var handler = events[name].bind(elem);
+    bindEvent(elem, name, function (e) {
+      if (ready) {
+        handler(e);
+      } else {
+        queue.push(handler.bind(elem, e));
```

Comment:
Is there any problem with calling `bind` twice on the original `events[name]` here? It seems OK to me (it's just that the original `this` can't be overridden it is set to the same object in both calls).